TrbMultiTool/TrbMultiTool/FileFormats/PProperty.cs

```diff
@@ -124,8 +121,14 @@ private TypeContent Player(ref TreeViewItem prev)
             var playerSubInfos = new List<SubInfo>();
             for (int i = 0; i < playerInfo.SubInfoOffsetCount; i++)
             {
-                playerSubInfos.Add(new SubInfo((Type)Trb.SectFile.ReadUInt32(), Trb.SectFile.ReadUInt32()));
+                Type type = (Type)Trb.SectFile.ReadUInt32();
+                uint valuePointer = (uint)Trb.SectFile.BaseStream.Position;
```

Comment:
You could just do something like this:
playerSubInfos.Add(new SubInfo((Type)Trb.SectFile.ReadUInt32(), Trb.SectFile.ReadUInt32(), (uint)Trb.SectFile.BaseStream.Position - 4));